RoutingStrategies.xaml

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--!
+    Name: Carissa Engebose, Pachia Lee, and Hannah Hotchkiss
+    Date: 10/16/24
+    Description: This implements and displays the routing strategies screen for the airports lab.
+                 It displays the starting airports, max distance, number of airports visited and 
+                 suggested routes as well as a map.
+    Bugs: The back button that was on the screen isn't there right now because navigation 
+          to this page has to come from planning tools. Also, we couldn't get the map to display 
+          a location, but the map is there.
+    Reflection: Overall this lab was very useful to learn about tabs and how to get a map for an app. 
+                There were a lot of difficulties especially with the map not working how we wanted it to, 
+                but it was nice to get practice on tabbed pages and work with new people.
+-->
+<TabbedPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
```

Comment:
I just realized this ... you've put your page in a TabbedPage, but that's already been done for you. If you look at MainTabbedPage, it lists your page as one of its children. So lines 15 to 23 are unnecessary,